test/fixtures/custom-manifest.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
we should make these obvious fixture ones the module-replacements repo will never have

otherwise the tests will probably be passing just because the MR manifests already contain these modules

e.g. just have 2-3 fake modules here like `@e18e/fake-0`, `@e18e/fake-1`, etc

and assert against that

Reply:
ok, let me handle this 🤘🏻